src/main/webapp/app/exercises/shared/example-submission/example-submissions.component.ts

```diff
@@ -69,4 +83,25 @@ export class ExampleSubmissionsComponent implements OnInit {
             ];
         }
     }
+
+    /**
+     * Opens the import module for example submission
+     * Then invokes import api for selected submission
+     */
+    openImportModal() {
+        const exampleSubmissionImportModalRef = this.modalService.open(ExampleSubmissionImportComponent, {
+            size: 'lg',
+            backdrop: 'static',
+        });
+        exampleSubmissionImportModalRef.componentInstance.exercise = this.exercise;
+        exampleSubmissionImportModalRef.result.then((selectedSubmission: Submission) => {
+            this.exampleSubmissionService.import(selectedSubmission.id!, this.exercise.id!).subscribe(
+                () => {
+                    this.alertService.success('artemisApp.exampleSubmission.submitSuccessful');
+                    location.reload();
```

Comment:
```suggestion
                    this.ngOnInit();
```
might also work, according to https://stackoverflow.com/a/51453244/11948496